cli/cmd/env/show.go

```diff
@@ -28,14 +28,20 @@ func ShowCmd(ch *cmdutil.Helper) *cobra.Command {
 			}
 
 			resp, err := client.GetProjectVariables(cmd.Context(), &adminv1.GetProjectVariablesRequest{
-				OrganizationName: ch.Org,
-				Name:             projectName,
+				Organization: ch.Org,
+				Project:      projectName,
+				Environment:  environment,
 			})
 			if err != nil {
 				return err
 			}
 
-			res, err := godotenv.Marshal(resp.Variables)
+			vars := make(map[string]string)
+			for _, v := range resp.Variables {
+				vars[v.Name] = v.Value
+			}
+
+			res, err := godotenv.Marshal(vars)
```

Comment:
I think if `--environment` is not passed, it should list all variable values for all environments (since that's what the UI does). So this probably needs to be replaced with a table that shows name, value and environment.